bedrock/mozorg/helpers/misc.py

```diff
@@ -365,33 +365,31 @@ def donate_url(ctx, source=''):
 
     For en-US this would output:
 
-        https://sendto.mozilla.org/page/contribute/EOYFR2013-tabzilla
+        https://donate.mozilla.org/en-US/?presets=100,50,25,15&amount=50&ref=EOYFR2015&utm_campaign=EOYFR2015&utm_source=mozilla.org&utm_medium=referral&utm_content=footer&currency=usd
 
     For de this would output:
 
-        https://sendto.mozilla.org/page/contribute/EOYFR2013-webDE
-
-    For fr this would output:
+        https://donate.mozilla.org/de/?presets=100,50,25,15&amount=50&ref=EOYFR2015&utm_campaign=EOYFR2015&utm_source=mozilla.org&utm_medium=referral&utm_content=footer&currency=eur
 
-        https://sendto.mozilla.org/page/contribute/EOYFR2013-webFR
+    For a locale not defined in settings.DONATE this would output:
 
-    For pt-BR this would output:
-
-        https://sendto.mozilla.org/page/contribute/EOYFR2013-webPTBR
+        https://donate.mozilla.org/ca/?presets=100,50,25,15&amount=50&ref=EOYFR2015&utm_campaign=EOYFR2015&utm_source=mozilla.org&utm_medium=referral&utm_content=footer&currency=usd
 
     """
     locale = getattr(ctx['request'], 'locale', 'en-US')
-    if locale not in settings.DONATE_LOCALE_LINK:
-        if 'default' in settings.DONATE_LOCALE_LINK:
-            locale = 'default'
-        else:
-            locale = 'en-US'
+    index = locale
+
+    # if the current locale is not available in settings.DONATE,
+    # we want to fallback to the default params but, still pass
+    # the current locale in the URL
```

Comment:
Maybe add a note why we're doing it (waiting for donate.mozilla.org to fix the issue with locale detection).